modules/apps/frontend-theme/frontend-theme-admin/src/css/_clay_variables.scss

```diff
@@ -23,6 +23,7 @@ $compat-buttons: false;
 $compat-cards: false;
 $compat-figures: false;
 $compat-form_validation: false;
+$compat-grid: false;
```

Comment:
Let's do more than this, let's **remove** it from `styled` so we don't need to do it here. See my [comments](https://issues.liferay.com/browse/LPS-102092?focusedCommentId=2314006&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-2314006) about the real goal of these tasks.

Reply:
Ok !!!!

Reply:
https://github.com/liferay-frontend/liferay-portal/pull/499/commits/4a3843ef20c557f81bc6737b1d35469475df81d6